### PR TITLE
✨ Add items that are supposed to be collected

### LINKED ISSUE #32: Add items that are supposed to be collected
## Acceptance Criteria ##

- The items are displayed on the pavement.
- Niffler can collect these items.

### PR DESCRIPTION
**Pull request by cboenke**
Closes #32 

- Some items are placed randomly on the pavement.
- Not working yet: avoid placement where Niffler can not go (granny/purse)

### CODE REVIEW

**Review by fre-ben — APPROVED** [review]
LGTM 🤙

File: components/game/Game.tsx [hunk 1]
```diff
@@ -30,13 +35,55 @@ function Game() {
...
         frameHeight: 51,
       });
+      this.load.image("coin", "/coin.svg");
+      this.load.image("jewel", "/jewel.svg");
+      this.load.image("twistedBracelet", "/twistedBracelet.svg");
+      this.load.image("jewelBracelet", "/jewelBracelet.svg");
+      this.load.image("necklace", "/necklace.svg");
+      this.load.image("ruby", "/ruby.svg");
+      this.load.image("emerald", "/emerald.svg");
+      this.load.image("chalice", "/chalice.svg");
```
Comment (lmachens) [inline components/game/Game.tsx]: You can reduce this if you make use of an array:
```suggestion
      // Array is not complete
      ['coin', 'jewel', 'twistedBracelet', 'jewelBracelet'].forEach((imageName) => {
            this.load.image(imageName, `/${imageName}.svg`);
      });
```

File: components/game/Game.tsx [hunk 2]
```diff
@@ -30,13 +35,55 @@ function Game() {
...
       granny = this.physics.add.image(1900, 140, "granny").setImmovable();
       this.physics.add.image(1935, 224, "purseNoClasp");
       purseClasp = this.physics.add.image(1935, 224, "purseClasp");
+      necklace = this.physics.add.image(1320, 165, "necklace");
+      chalice = this.physics.add.image(
+        2500,
+        Phaser.Math.Between(165, 350),
+        "chalice"
+      );
+      randomTreasures = this.randomTreasures = [
```
Comment (lmachens) [inline components/game/Game.tsx]: Do you need `this.randomTreasures`?

File: components/game/Game.tsx [hunk 3]
```diff
@@ -30,13 +35,55 @@ function Game() {
...
+        key: randomTreasures,
+        frameQuantity: 1,
+      });
+
+      const children = collectGroup.getChildren();
+
+      for (let i = 0; i < children.length; i++) {
+        const randomX = Phaser.Math.Between(150, 2300);
+        const randomY = Phaser.Math.Between(165, 350);
+        console.log(randomY);
```
Comment (lmachens) [inline components/game/Game.tsx]:
```suggestion
```

File: components/game/Game.tsx [hunk 4]
```diff
@@ -93,6 +153,18 @@ function Game() {
...
+      children.disableBody(true, true);
+    }
+
+    function touchNecklace(necklace) {
+      necklace.disableBody(true, true);
+    }
+
+    function touchChalice(chalice) {
+      chalice.disableBody(true, true);
+    }
```
Comment (lmachens) [inline components/game/Game.tsx:169]: Right now, these functions are very similar. I expect that you add differences later (like different points depending on treasure)?
If not, I would reduce it to a single function.

**Review by lmachens — COMMENTED** [review]
Good job :).

I would love to see type definitions and outsourced game logic in upcoming changes 👍

**Review by mewiemers — COMMENTED** [review]
GJ!🐁

**Review by sweisz — APPROVED** [review]
GJ 👾

**Pull request merged**